english/dig/dig.md

```diff
@@ -8,7 +8,7 @@ It says "hello"
 
 "Hello, Mr Bird!", you say, and spot a collar with a tag on it.
 
-Upon inspecting the tag, you see it reads: "Nevermore". Turning over the tag, you see: "If found, please return to:
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.